### ISSUE TITLE
Default login on newly created installation

### ISSUE THREAD

**Issue by mnellemann** [KETSE/casebox#52]
Hi,

I've got Casebox (the new version from git) installed, but I can't find any information about the default admin login. Is this available somewhere or did I miss steps during installation ?

**Comment by vladimir-m** [comment]
Hi @mnellemann 
Casebox default username and password: root / a

**Comment by ben-apprio** [comment]
I am running CaseBox through vagrant.  The default login of **root** / **a** does not work for me.  I also tried root/test from the online demo, but that didn't work either.

**Comment by vmelnic** [comment]
Hi @ben-apprio 
What password you put into the `Casebox root password` field? http://storage4.static.itmages.com/i/16/1006/h_1475777133_8422602_03e57d443c.png

You can update/change root password by using console command 
`php /var/www/casebox/bin/console casebox:user:password:update --help root StrongPassword`
http://storage8.static.itmages.com/i/16/1006/h_1475777803_3522497_78d38d3202.png

**Comment by ben-apprio** [comment]
Yeah - I figured that out.   I am apparently running a different version than mnellemann.
